whirlpool/eventsocket.py

```diff
@@ -45,12 +67,12 @@ async def _recv_msg(self, websocket: aiohttp.ClientWebSocketResponse):
     async def _run(self):
         if not self._running:
             return
-
-        timeout = aiohttp.ClientTimeout(total=None)
+        await self._fetch_data()
```

Comment:
put this after after connection is done and subscribed, since it is only there when the data starts coming from the WS